`//MODULE ` with a trailing space causes an error

**Describe the bug**
jbang-vscode provides completion for `//MODULE`, but it inserts a space at the end, so the user can type the module name directly. However, if you don't specify a custom module name, JBang will error with:
```
[jbang] [ERROR] //MODULE line has wrong format, should be '//MODULE [identifier]'
```

**To Reproduce**
Steps to reproduce the behavior:
1. Create a JBang file in a package, eg. foo/Hello.java
2. Add the `//MODULE ` directive (with trailing space(s))
3. Run `jbang foo/Hello.java` or `jbang info tools foo/Hello.java`
4. See `[jbang] [ERROR] //MODULE line has wrong format, should be '//MODULE [identifier]'` error

**Expected behavior**
Since `//MODULE` defaults to an automatic module name, JBang should be more robust and trim the `//MODULE` directive  line before causing an error.

**JBang version**
0.105.2

--- a/src/main/java/dev/jbang/source/TagReader.java
+++ b/src/main/java/dev/jbang/source/TagReader.java
@@ -32,9 +32,6 @@ public abstract class TagReader {
 	protected final String contents;
 	protected final Function<String, String> replaceProperties;
 
-	// Cached values
-	private List<String> lines;
-
 	private static final String REPOS_COMMENT_PREFIX = "REPOS ";
 	private static final String DEPS_COMMENT_PREFIX = "DEPS ";
 	private static final String FILES_COMMENT_PREFIX = "FILES ";
@@ -168,6 +165,7 @@ public abstract class TagReader {
 	public Optional<String> getModule() {
 		List<String> mods = getTags()
 										.filter(this::isModuleDeclare)
+										.map(String::trim)
 										.map(s -> s.substring(MODULE_COMMENT_PREFIX.length()))
 										.collect(Collectors.toList());
 		if (mods.isEmpty()) {

--- a/src/test/java/dev/jbang/source/TestModule.java
+++ b/src/test/java/dev/jbang/source/TestModule.java
@@ -116,8 +116,17 @@ public class TestModule extends BaseTest {
 
 	@Test
 	void testEmptyModule(@TempDir File output) throws IOException {
+		testEmptyModule(srcWithEmptyModDep, output);
+	}
+
+	@Test
+	void testEmptyModuleTrailingWhiteSpaces(@TempDir File output) throws IOException {
+		testEmptyModule(srcWithEmptyModDep.replace("//MODULE", "//MODULE  \t  "), output);
+	}
+
+	void testEmptyModule(String script, File output) throws IOException {
 		Path f = output.toPath().resolve("moduletest.java");
-		Util.writeString(f, srcWithEmptyModDep);
+		Util.writeString(f, script);
 
 		ProjectBuilder pb = Project.builder();
 		Project prj = pb.build(f);

